integration/apps/minife/0005-Add-region-markup.patch

```diff
@@ -105,20 +152,32 @@ index cdf9b15..4f54069 100644
    if(beta == 0.0) {
  	if(alpha == 1.0) {
    		#pragma omp parallel for
-@@ -177,6 +188,9 @@ void
+@@ -177,6 +189,9 @@ void
    		}
    	}
    }
 +#ifndef NO_GEOPM
 +  geopm_prof_exit( R_ID);
 +#endif
-
+ 
  #ifdef MINIFE_DEBUG_OPENMP
    std::cout << "Finished WAXPBY." << std::endl;
-@@ -245,10 +259,20 @@ typename TypeTraits<typename Vector::ScalarType>::magnitude_type
+@@ -185,9 +200,9 @@ void
+ 
+ template<typename VectorType>
+ void
+-  daxpby(const MINIFE_SCALAR alpha, 
++  daxpby(const MINIFE_SCALAR alpha,
+ 	const VectorType& x,
+-	const MINIFE_SCALAR beta, 
++	const MINIFE_SCALAR beta,
+ 	VectorType& y)
+ {
```

Comment:
What is this change about?

Reply:
Looks like an automated wsp trimming ", " vs ","

Reply:
If you want to address wsp issues, please pull that into a standalone patch like I did with nekbone/0001-Fix-whitespace-issues.patch.  I'm also fine if you just want to remove this change.  It has nothing to do with adding region markup.